apps/mull-ui-e2e/src/integration/US-1.1.spec.ts

```diff
@@ -132,9 +148,14 @@ frameSizes.forEach((frame) => {
     it('should show a successful submission message', () => {
       cy.get('#imageFile').attachFile('../fixtures/trashed-park.jpg');
 
-      cy.get('#startTime').type('11:20');
-
-      cy.get('#endTime').type('15:20');
+      cy.get('.rc-slider-handle')
+        .first()
+        .should('have.attr', 'aria-valuenow', 12)
+        .type('{rightarrow}');
+      cy.get('.rc-slider-handle')
+        .last()
+        .should('have.attr', 'aria-valuenow', 12)
+        .type('{leftarrow}');
```

Comment:
This fixes the e2e test that is failing uniquely for this ticket
```suggestion
      cy.get('.rc-slider-handle')
        .first()
        .should('have.attr', 'aria-valuenow', 12)
        .type('{leftarrow}');
      cy.get('.rc-slider-handle')
        .last()
        .should('have.attr', 'aria-valuenow', 12)
        .type('{rightarrow}');
```